src/ui/molecules/ToggleComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactNode } from 'react';
+import { ContentToggle, SelectField } from '../atoms';
+
+export interface Value {
+    label: string;
+    value: string;
+}
+
+interface Props {
```

Comment:
Have a think, are all these properties needed? Are there any that are only internal to the component and perhaps can be set another way?